src/Nixfmt/Types.hs

```diff
@@ -424,6 +424,8 @@ data Token
   | TLessEqual
   | TNot
   | TUnequal
+  | TPipeTo
+  | TPipeFrom
```

Comment:
Small nix: I'd prefer PipeRight and PipeLeft (or PipeForward/PipeBackwards) as names, as they're a bit more visual.

Reply:
Done by @dasJ in https://github.com/NixOS/nixfmt/pull/227/commits/55feeb57b09c8f2df363e78e9842bf5d0c648936